reddit (1).py

```diff
@@ -0,0 +1,149 @@
+import praw
```

Comment:
remove un-needed files, or make a directory that contains work related to making a map of a reddit comment thread.